### ISSUE TITLE
XRefs to DLEntry Elements Do Not Make Working Links

### ISSUE THREAD

**Issue by drmacro** [dita-ot/dita-ot#1764]
Given a DL entry like:

~~~~
      <dlentry
        id="def-cherry">
        <dt>cherry</dt>
        <dd>A smaller red fruit.</dd>
      </dlentry>
~~~~

And an xref like:

~~~~
<xref href="#topicid/def-cherry"/>
~~~~

The 1.8.5 PDF2 transform correctly reflects the DT text in the link but the link itself is not resolved to the anchor (reflects page 0 and does not take you to the DL entry.

The same link works correctly in HTML.

I have a small test case that I can provide. 

I will look into the code and see if I work up a fix.

**Comment by robander** [comment]
Looks like a duplicate of #2198 which has a fix coming up in 2.2.5 (needed for FOP only, guessing maybe AH has fixed the issue on their side since this was opened in 2014).

**Comment by infotexture** [comment]
Looks like we should add the **2.2.5** milestone to this.
